src/metabase/models/setting.clj

```diff
@@ -521,12 +526,14 @@
 
      (mandrill-api-key \"xyz123\")"
   [setting-definition-or-name new-value]
-  (let [{:keys [setter cache?], :as setting} (resolve-setting setting-definition-or-name)
+  (let [{:keys [setter type cache?], :as setting} (resolve-setting setting-definition-or-name)
         name                                 (setting-name setting)]
```

Comment:
This is a super minor nitpick but if the `let` bindings were aligned it's nice to keep them aligned. Or just switch them to unaligned if you don't have the editor set up to do it for you -- that's fine too